plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMinioOrcConnectorTest.java

```diff
@@ -135,6 +135,25 @@ private void testReadSingleIntegerColumnOrcFile(String orcFileResourceName, int
         }
     }
 
+    @Test
+    public void testTimeType()
```

Comment:
Anyway, what is this test testing? what's the role of `single-time-column.orc` file?
How was `single-time-column.orc` produced?

Reply:
We can reproduce the issue without `single-time-column.orc`
```sql
CREATE TABLE test (c1 time(6));
INSERT INTO test VALUES (TIME '13:30:00'), (TIME '14:30:00'), (NULL);
SELECT * FROM test;
-- Unsupported type time(6)
```